src/celestia/qt/qtappwin.cpp

```diff
@@ -92,8 +92,8 @@ static const int CELESTIA_MAIN_WINDOW_VERSION = 12;
 static CelestiaAppWindow* MainWindowInstance = nullptr;
 static void ContextMenu(float x, float y, Selection sel);
 
-static int fps_to_ms(int fps) { return fps ? 1000 / fps : 0; }
-static int ms_to_fps(int ms) { return ms ? 1000 / ms : 0; }
+static int fps_to_ms(int fps) { return fps > 0 ? 1000 / fps : 0; }
```

Comment:
Is it possible to use mks instead of ms? Then you'll have a more correct result.

Reply:
It's impossible with qt timers. If c++ std library offer more accurate timer, then maybe...